pyface/ui/wx/color_dialog.py

```diff
@@ -0,0 +1,63 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+""" The interface for a dialog that allows the user to select a color. """
+
+import wx
+
+from traits.api import Bool, provides
+
+from pyface.color import Color, PyfaceColor
+from pyface.i_color_dialog import IColorDialog
+from .dialog import Dialog
+
+
+@provides(IColorDialog)
+class ColorDialog(Dialog):
+    """ A dialog for selecting colors.
+    """
+
+    # 'IColorDialog' interface ----------------------------------------------
+
+    #: The color in the dialog.
+    color = PyfaceColor()
+
+    #: Whether or not to allow the user to chose an alpha value.
+    show_alpha = Bool(False)
+
+    # ------------------------------------------------------------------------
+    # 'IDialog' interface.
+    # ------------------------------------------------------------------------
+
+    def _create_contents(self, parent):
+        # In wx this is a canned dialog.
+        pass
+
+    # ------------------------------------------------------------------------
+    # 'IWindow' interface.
+    # ------------------------------------------------------------------------
+
+    def close(self):
+        color_data = self.control.GetColorData()
+        wx_color = color_data.GetColor()
+        self.color = Color.from_toolkit(wx_color)
+        super(ColorDialog, self).close()
+
+    # ------------------------------------------------------------------------
+    # 'IWidget' interface.
+    # ------------------------------------------------------------------------
+
+    def _create_control(self, parent):
+        wx_color = self.color.to_toolkit()
+        data = wx.ColorData()
```

Comment:
Trying to open a dialog in wx fails with error:
```
>>> get_color(None, "red")

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/icernyte/Documents/repos/pyface/pyface/color_dialog.py", line 39, in get_color
    result = dialog.open()
  File "/Users/icernyte/Documents/repos/pyface/pyface/i_dialog.py", line 164, in open
    self._create()
  File "/Users/icernyte/Documents/repos/pyface/pyface/i_dialog.py", line 183, in _create
    super(MDialog, self)._create()
  File "/Users/icernyte/Documents/repos/pyface/pyface/i_widget.py", line 108, in _create
    self.control = self._create_control(self.parent)
  File "/Users/icernyte/Documents/repos/pyface/pyface/ui/wx/color_dialog.py", line 59, in _create_control
    data = wx.ColorData()
AttributeError: module 'wx' has no attribute 'ColorData'
```

Reply:
WxPython uses UK spellings!

Reply:
Opening the dialog still fails, but now with a different error on the next line:
```
>>> get_color(None, "red")

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/icernyte/Documents/repos/pyface/pyface/color_dialog.py", line 39, in get_color
    result = dialog.open()
  File "/Users/icernyte/Documents/repos/pyface/pyface/i_dialog.py", line 164, in open
    self._create()
  File "/Users/icernyte/Documents/repos/pyface/pyface/i_dialog.py", line 183, in _create
    super(MDialog, self)._create()
  File "/Users/icernyte/Documents/repos/pyface/pyface/i_widget.py", line 108, in _create
    self.control = self._create_control(self.parent)
  File "/Users/icernyte/Documents/repos/pyface/pyface/ui/wx/color_dialog.py", line 60, in _create_control
    data.SetInitialColour(wx_colour)
AttributeError: 'ColourData' object has no attribute 'SetInitialColour'
```